tests/dummy/app/templates/demo/grid-list.hbs

```diff
@@ -1,49 +1,50 @@
-{{page-toolbar pageTitle="Grid List" isDemo=true isWIP=true}}
+{{page-toolbar pageTitle="Grid List" isDemo=true}}
 
 {{#doc-content}}
   <h2>Basic Usage</h2>
   {{! BEGIN-SNIPPET grid-list.basic-usage }}
   {{#paper-content class="md-whiteframe-z1 grid-list-demo1"}}
 
-    {{#paper-grid-list md-cols-sm="1" md-cols-md="2" md-cols-gt-md="6"
-    md-row-height-gt-md="1:1" md-row-height="2:2"
-    md-gutter="12px" md-gutter-gt-sm="8px"}}
-
-      {{#paper-grid-tile class="gray" md-rowspan="3" md-colspan="2" md-colspan-sm="1"}}
-        {{#paper-grid-tile-footer}}
-            <h3>#1: (3r x 2c)</h3>
-        {{/paper-grid-tile-footer}}
-      {{/paper-grid-tile}}
-
-      {{#paper-grid-tile class="green"}}
-        {{#paper-grid-tile-footer}}
-            <h3>#2: (1r x 1c)</h3>
-        {{/paper-grid-tile-footer}}
-      {{/paper-grid-tile}}
-
-      {{#paper-grid-tile class="yellow"}}
-        {{#paper-grid-tile-footer}}
-            <h3>#3: (1r x 1c)</h3>
-        {{/paper-grid-tile-footer}}
-      {{/paper-grid-tile}}
-
-      {{#paper-grid-tile class="blue" md-rowspan="2"}}
-        {{#paper-grid-tile-footer}}
+    {{#paper-grid-list
+      cols-sm="1" cols-md="2" cols-gt-md="6"
```

Comment:
Did you keep dash-cased attributes here on purpose? It is true that size prefixes (`-sm`, `-xs`, `-lg`, etc) usually are dash-cased. However, I think consistency wins here. Trying to think of a good api for this.

Let me know what you think.